engine/release-notes/index.md

```diff
@@ -15,6 +15,35 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.22
+{% include release-date.html date="2022-12-16" %}
+
+This release of Docker Engine contains updated versions of Docker Compose,
+Docker Scan, Containerd, and some minor bug fixes and enhancements.
+
+### Updates
+
+- Update Docker Compose to [v2.14.1](https://github.com/docker/compose/releases/tag/v2.14.1){:target="_blank" rel="noopener"}.
+- Update Docker Scan to [v0.23.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.23.0){:target="_blank" rel="noopener"}.
+- Update containerd (`containerd.io` package) to [v1.6.13](https://github.com/containerd/containerd/releases/tag/v1.6.13){:target="_blank" rel="noopener"},
+  includes fixes prior (1.6.9, 1.6.10, 1.6.11, and 1.6.12) patch releases, which
+  include a fix for [CVE-2022-23471](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23471){:target="_blank" rel="noopener"}.
+- Update Go runtime to [1.18.9](https://go.dev/doc/devel/release#go1.18.minor){:target="_blank" rel="noopener"},
+  includes updates from prior (1.18.8 and 1.18.9) releases, which include fixes for
```

Comment:
Can we remove the middle bit? Makes it easier to read, and I don't think we lose any relevant details.

```suggestion
  which includes fixes for
```

Reply:
Ah, yes, that would work; I changed it to "to include fixes for" (for both Go and containerd); it felt less ambiguous (i.e., the latest version itself doesn't have the fix, but "update to `<latest version>` to include fixes for `<vulns>`" doesn't try to implicate that the latest version has the actual fix.

Happy to change to "which" though!